### ISSUE TITLE
Tweak kubernetes-external-secrets with gsm - causing a pretty decent cost

### ISSUE THREAD

**Issue by sadym** [jenkins-x/jx3-versions#2417]
(hopefully I got the right repo for this issue)

I think the default 10sec interval is a little bit too much for a default setup, as keys are probably changed once a week at most. It is also causing a pretty decent cost in gsm, with no more than 100 keys (probably using `latest` instead of specific version adds to that also). Indeed, it can be easily customized per installation, but I think a 60s value would not affect the installation (when externalsecrets are created, the secrets are populated immediately).
Not mentioning the logging volume. I attached a screenshot for 3 hours interval with default LOG_LEVEL=info (this should probably be an issue for kubernetes-external-secret project).  
 
![image](https://user-images.githubusercontent.com/1901393/115837910-c6bbbb00-a421-11eb-882c-0467305a3e17.png)

![image](https://user-images.githubusercontent.com/1901393/115839462-59108e80-a423-11eb-84a2-4528a8cc7f8d.png)

**Comment by sadym** [comment]
I've seen that the may LTS release suppose to have "reduced external secrets polling", but to me it looks that the change applies only to vault:
https://github.com/jenkins-x/jx3-versions/blob/master/charts/external-secrets/kubernetes-external-secrets/values.yaml.gotmpl#L19 

```{{- if eq .Values.jxRequirements.secretStorage "vault" }}
env:
  VAULT_ADDR: https://vault.jx-vault:8200
  NODE_EXTRA_CA_CERTS: "/usr/local/share/ca-certificates/ca.crt"
  POLLER_INTERVAL_MILLISECONDS: 60000
```

**Comment by sadym** [comment]
@rawlingsj, #2417 seems to be only about vault. Is there another place that overrides it that I'm missing?

**Comment by sadym** [comment]
seems to be fixed with #2557
